packages/page-supersig/src/Supersig/Dashboard/index.tsx

```diff
@@ -0,0 +1,146 @@
+// Copyright 2017-2022 @polkadot/app-addresses authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ComponentProps as Props } from '../types';
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import { Table } from '@polkadot/react-components';
+import { useApi, useCall, useFavorites, useToggle } from '@polkadot/react-hooks';
+import { useLoadingDelay } from './useLoadingDelay'
+import CreateModal from '../modals/Create';
+import { useTranslation } from '../translate';
+import Address from './Table';
+import Summary from './Summary';
+import { largeNumSum } from '../../util';
+import { encodeAddress } from '@polkadot/util-crypto';
+
+type SortedAddress = { address: string; isFavorite: boolean };
+
+const STORE_FAVS = 'accounts:favorites';

```

Comment:
This should not be account favorites - they are only managed in the accounts/addresses sections.